pom.xml

```diff
@@ -46,23 +46,21 @@
     <dependency>
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jsr310</artifactId>
+      <version>2.13.4</version>
     </dependency>
     <!-- TEST -->
     <dependency>
       <groupId>org.springframework.cloud</groupId>
       <artifactId>spring-cloud-stream</artifactId>
+      <version>3.2.5</version>
       <scope>test</scope>
       <classifier>test-binder</classifier>
       <type>test-jar</type>
     </dependency>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.kafka</groupId>
-      <artifactId>spring-kafka-test</artifactId>
+      <version>2.7.3</version>
```

Comment:
There is no need to explicit the version here. Upgrade the spring-boot-starter-parent at the top page of pom.xml and all dependencies contained will be upgraded consequently (if no version is specified). 

See here https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-parent/2.7.3. 

As you can see the spring-boot version 2.7.3 already contains the spring-core 5.3.22. You may gain in maintainability and compatibility if you just inherit dependencies version from the parent.